tests/config_spec.vim

```diff
@@ -48,12 +49,14 @@ onedarkpro.setup({
         },
         TestHighlightGroup = {
             fg = "${red}",
-            style = "italic"
+            italic = true
```

Comment:
Can we revert this to `style = "italic"`. We still need to make sure that this is tested for existing users.